.travis.yml

```diff
@@ -1,16 +1,13 @@
 # Copyright 2016, 2017 Peter Dimov
+# Copyright 2021 Matt Borland
 # Distributed under the Boost Software License, Version 1.0.
 # (See accompanying file LICENSE_1_0.txt or copy at http://boost.org/LICENSE_1_0.txt)
 
 language: cpp
 
 sudo: false
 
-python: "2.7"
-
-os:
-  - linux
-  - osx
+dist: focal 
```

Comment:
@ckormanyos Here is where you would plug in the distro that you want to use. The [Travis docs](https://docs.travis-ci.com/user/reference/linux) show the versions of the default tools in the toolchain by OS. Focal is the latest LTS version of Ubuntu.